problemsets-mat223/modules/module14.tex

```diff
@@ -266,7 +273,15 @@
 \begin{example}
 	Use elementary matrices to find the determinant of $A=\mat{1&2\\3&4}$.
 
-	XXX Finish
+	By applying row reduction to $A$, we could obtain
```

Comment:
-> `We can row-reduce $A$ with the following steps`. Spell out the steps, then do the factorization.